extensions.json

```diff
@@ -68,15 +83,30 @@
     "repository": "https://github.com/apollographql/apollo-midnight"
   },
   "appulate.filewatcher": {
-    "repository": "https://github.com/appulate/vscode-file-watcher"
+    "repository": "https://github.com/appulate/vscode-file-watcher",
+    "prepublish": "[ $VERSION = '1.0.3' ] && sed -i 's/\\(: \\)\\(vscode.WorkspaceFolder\\[\\]\\)/\\1readonly \\2/g' src/extension.ts || true"
   },
   "arcanis.vscode-zipfs": {
     "repository": "https://github.com/yarnpkg/berry",
     "location": "packages/vscode-zipfs",
     "prepublish": "sed -ri \"s_ssh://git@github.com/yarnpkg/berry.git_https://github.com/yarnpkg/berry_g\" packages/vscode-zipfs/package.json"
   },
   "arjun.swagger-viewer": {
-    "repository": "https://github.com/arjun-g/vs-swagger-viewer"
+    "repository": "https://github.com/arjun-g/vs-swagger-viewer",
+    "location": ".",
+    "custom": [
+      "[ $VERSION = '3.1.2' ] && sed -i '/\"@types\\/socket\\.io\": \".*\"/d' package.json || true",
```

Comment:
I'm thinking about whether it is necessary here, because `sed` could theoretically replace these values on any version of the extension, and if it is fixed, sed would just continue without replacing, correct?

I am just a bit worried about a lot of maintenance after these extensions get updates and a lot of cleanups necessary afterwards. If we only want to do some fixes for one version only, we could run the publish-extensions workflow on this branch one-time so that those versions are all published, making the fixes being in `master` unnecesary.

Reply:
That is correct. A command which is prefixed with such a version guard will only execute if the extension has a specific version.
But I think running the publish task and cleaning up the `extensions.json` file before merging sounds like a great idea 😄 

Are there any version-guarded commands that should remain in the `extension.json` file?
I think keeping the one of `serayuzgur.crates` inside might make sense as the `package.json` file in this repository isn't properly versioned and there are no git tags either causing the publish-workflow to `git checkout` any random tag which is contained in the `msVersion` name (by coincidence)